crates/rover-client/src/operations/subgraph/publish/runner.rs

```diff
@@ -119,8 +77,8 @@ fn build_response(publish_response: UpdateResponse) -> SubgraphPublishResponse {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
     use serde_json::json;
+    use super::*;
```

Comment:
This change is causing a lint failure.
```suggestion
    use super::*;
    use serde_json::json;
```

Reply:
Resolved by lint